src/test/ui/lint/cli-lint-override.forbid_warn.stderr

```diff
@@ -2,7 +2,7 @@ error: extern declarations without an explicit ABI are deprecated
   --> $DIR/cli-lint-override.rs:12:1
    |
 LL | extern fn foo() {}
-   | ^^^^^^^^^^^^^^^ ABI should be specified here
+   | ^^^^^^^^^^^^^^^^^^ ABI should be specified here
```

Comment:
This will cause the entire function to be marked in ide's instead of just the function signature, which is annoying.